.travis.yml

```diff
@@ -19,6 +19,7 @@ cache:
 
 addons:
   apt:
+    update: true
```

Comment:
wild guess to prevent `travis_apt_get_update` error (or move the error to the beginning of the build?)